infra/main.bicep

```diff
@@ -21,13 +21,15 @@ param searchServiceName string = '' // Set in main.parameters.json
 param searchServiceResourceGroupName string = '' // Set in main.parameters.json
 param searchServiceLocation string = '' // Set in main.parameters.json
 // The free tier does not support managed identity (required) or semantic search (optional)
-@allowed([ 'free', 'basic', 'standard', 'standard2', 'standard3', 'storage_optimized_l1', 'storage_optimized_l2' ])
+@allowed(['free', 'basic', 'standard', 'standard2', 'standard3', 'storage_optimized_l1', 'storage_optimized_l2'])
```

Comment:
Are all these formatting changes because we automatically formatted the file?

Reply:
That's my assumption, that Yefu ran az bicep format. We could revert the formatting changes to make this change easier to parse, and apply them later. Ideally we'd always format but I guess I dont check that in CI yet.

Reply:
Yes, I ran the `Format Document` in vscode. Does it need to be reverted?

Reply:
I merged a change in the main branch to run the formatting, and then merged that change into this branch, so now it only has your changes, not the other formatting changes. So that's not an issue any longer.